pyasic/miners/backends/innosilicon.py

```diff
@@ -90,6 +91,10 @@
             "_get_uptime",
             [RPCAPICommand("rpc_stats", "stats")],
         ),
+        str(DataOptions.POOLS): DataFunction(
+            "_get_pools",
+            [WebAPICommand("web_pools", "pools")]
```

Comment:
Prefer to use RPC endpoints wherever possible, RPC commands are much less network intensive VS web commands.  I also have a lot less info on what the Innosilicon web pools result looks like, I might have access to a T3H+ to test, but I'll have to see.